app/views/static_pages/splash.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="flex flex-col justify-between h-screen items-center">
+  <p class="text-center p-3 blue-btn w-full">Welcome!</br>to my awesome app :)</p>
+  <h1 class="text-4xl font-extrabold blue animate-scale">
+    Budget Monitor
+  </h1>
+  <div class="w-full">
+    <% if user_signed_in? %>
+      <div>
+        <a href="<%= categories_path %>" class="w-full">
+          <button class="blue-btn w-full p-4">Start Monitoring!</button>
+        </a>
+      </div>
```

Comment:
- [ ] Great initiative :clap: . The **Categories page**, however, should be the user's first page after logging in, meaning that it should be our home page, in accordance with the criteria. To avoid needing to click a button to access the categories page, please ensure that the root of the app links to the categories page. :ok_hand: 

### Requirement :point_down: 

![Screenshot from 2023-09-14 23-01-39](https://github.com/Salimer/Budget-app/assets/89228743/7b64b335-8d81-4202-9384-e6937675eb15)